editor/project_settings_editor.cpp

```diff
@@ -875,7 +871,7 @@ void ProjectSettingsEditor::_item_add() {
 
 	undo_redo->commit_action();
 
-	globals_editor->set_current_section(catname);
+	globals_editor->set_current_section(name.split("/")[0]);
```

Comment:
4.0 uses `name.get_slice("/", 1)` instead. What's better?

Reply:
`get_slice()` is better, the other one can crash if there's no result.

Reply:
It seems to me that `string.split("/")` returns a `Vector<String>` containing at least one element. Ok, I'll fix it.

Reply:
Ah you're right, both are fine then.